azure_pipelines/changes.txt

```diff
@@ -1,2 +1,3 @@
 1
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->